### PR TITLE
Add tests for various aliased/depricated/yum-compat commands (RhBug:1821524)

### PR DESCRIPTION
**Pull request by kontura**
https://bugzilla.redhat.com/show_bug.cgi?id=1821524

These are tests for https://github.com/rpm-software-management/dnf/pull/1611

### CODE REVIEW

File: dnf-behave-tests/features/yum-compat.feature [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      """
+
+
+Scenario: install using autoremove-n
+Given I successfully execute dnf with args "install vagare"
+ When I execute dnf with args "autoremove-n vagare"
+ Then the exit code is 0
+  And Transaction is following
+      | Action       | Package                       |
+      | remove       | labirinto-0:1.0-1.fc29.x86_64 |
```
Comment (pkratoch) [inline dnf-behave-tests/features/yum-compat.feature:105]: Due to commit https://github.com/rpm-software-management/ci-dnf-stack/commit/d57c06d077774b492c7f59b237f46fc4c4c3d1dc, the action for labirinto should be ``remove-unused``.

**Comment by pkratoch** [comment]
There is also ``install-nevra`` command that might deserve a test (it should fail if the argument is incomplete nevra).

Otherwise looks good.

**Comment by kontura** [comment]
I have added the test for `install-nevra`, also I checked and all of these commands have the variants, so there is for example `autoremove-n`, `autorotemove-na` and `autorotemove-nevra`, I could add tests for all of them if you think its necessary.

**Comment by pkratoch** [comment]
I think it's ok. The tests passed locally, so I am merging this.

**Pull request merged**